ui/app/components/auth/form/base.hbs

```diff
@@ -22,7 +22,7 @@
       @text="Sign in"
       @isFullWidth={{true}}
       type="submit"
-      class="has-top-margin-m has-bottom-margin-m"
+      class="has-top-margin-xl has-bottom-margin-xl"
```

Comment:
I kept accidentally clicking "Sign in with other methods" 🙃 so added some spacing